autoload/vimteractive.vim

```diff
@@ -213,3 +230,36 @@ function! vimteractive#connect(...)
     echom "Connected " . bufname("%") . " to " . l:bufname
 
 endfunction
+
+function! vimteractive#get_response()
+    let l:term_type = getbufvar(b:vimteractive_connected_term, "vimteractive_term_type")
+    if (l:term_type == 'sgpt')
```

Comment:
I think this could be made extensible (e.g. use a dict instead of an if chain) so users can define their own `get_response` functions.

Reply:
I think I've now achieved this in 64cfad0 -- let me know if that's what you were imagining.

Reply:
Yeah, that’s close to what I had in mind.